projects/cds/public_api.ts

```diff
@@ -2,6 +2,8 @@
  * Public API Surface for CDS
  */
 export * from './src/cds-models/index';
+export * from './src/cds.module';
+export * from './src/cms-components/index';
```

Comment:
we shouldn't expose things to the public until we're 100% sure. We should include this in the final merge to have room for improvements as if we include it now, we can't change function names and such.

Reply:
We need to export it to be able to test it in the storefront application, no?

Reply:
we can get back to this before the last merge, but we would most likely want to expose the module  and configs only for now. But it's okay for now I guess